apps/balancer-tools/src/utils/fetcher.ts

```diff
@@ -5,9 +5,12 @@ export async function fetcher<JSON = unknown>(
   const response = await fetch(input, init);
 
   if (!response.ok) {
-    // eslint-disable-next-line no-console
-    console.error(await response.text());
-    throw new Error("Network response was not ok");
+    const errorBody = await response.json();
```

Comment:
could also check for in response.json() for `"error"` or `"errors"` key, since GQL queries that break won't return an non 2xx status but will come with those keys in the response